code/index.html

```diff
@@ -1,21 +1,44 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
+<head>
```

Comment:
Hi! Nice to meet you! I'm Estefanía and I'll be taking a look at your code today. 👋😀